data/selinux/snappy.te

```diff
@@ -160,6 +160,10 @@ optional_policy(`
   allow snappy_t dbusd_etc_t:lnk_file { read };
 ')
 
+# Allow snapd to manage kmod-backend (/etc/modules-load.d/) files
```

Comment:
Quite unfortunate that /etc labels aren't more granular, but there's nothing we can do.

Reply:
Yep..

Reply:
In this case, it may make sense to ask for the policy to become more granular here. A bug report requesting this to be more granular should be easy enough...